src/components/modal/modal-add-edit-content.tsx

```diff
@@ -43,169 +60,158 @@ const ModalAddEditContent = ({
+                  </MenuItem>
+                ))}
+              </Field>
+            </ModalGenres>
 
-        <TextField
-          label="Overview"
-          name="overview"
-          variant="filled"
-          color="secondary"
-          className={classes.formControl}
-          InputProps={{ className: classes.title }}
-          InputLabelProps={{
-            classes: {
-              root: classes.label,
-              focused: classes.focusedLabel,
-            },
-          }}
-          value={overview}
-          onChange={handleChangeOverview}
-        />
-        <TextField
-          label="Runtime"
-          name="runtime"
-          variant="filled"
-          color="secondary"
-          className={classes.formControl}
-          InputProps={{ className: classes.title }}
-          InputLabelProps={{
-            classes: {
-              root: classes.label,
-              focused: classes.focusedLabel,
-            },
-          }}
-          value={runtime}
-          onChange={handleChangeRuntime}
-        />
-      </>
-      <div className={classes.btns}>
-        <Button
-          variant="contained"
-          onClick={closeModal}
-          className={classes.btnSpace}
-        >
-          RESET
-        </Button>
-        <Button
-          variant="contained"
-          color="secondary"
-          onClick={(_) => {
-            onModalSubmit({
-              ...formData,
-              genres,
-              title,
-              overview,
-              poster_path: posterPath,
-              runtime,
-              release_date: releaseDate,
-            });
-          }}
-        >
-          SUBMIT
-        </Button>
-      </div>
-    </form>
+            <FormikTextField
+              label="Overview"
+              name="overview"
+              variant="filled"
+              color="secondary"
+              className={classes.formControl}
+              InputProps={{ className: classes.title }}
+              InputLabelProps={{
+                classes: {
+                  root: classes.label,
+                  focused: classes.focusedLabel,
+                },
+              }}
+              value={overview}
+              onChange={(e) => setOverview(e.target.value)}
+            />
+            <FormikTextField
+              label="Runtime"
+              name="runtime"
+              variant="filled"
+              color="secondary"
+              className={classes.formControl}
+              InputProps={{ className: classes.title }}
+              InputLabelProps={{
+                classes: {
+                  root: classes.label,
+                  focused: classes.focusedLabel,
+                },
+              }}
+              value={runtime}
+              onChange={(e) => setRuntime(+e.target.value)}
+            />
+          </>
+          <div className={classes.btns}>
+            <Button
+              variant="contained"
+              onClick={closeModal}
```

Comment:
Add `type="reset"` instead of `onClick` handler.

Reply:
It doesn't work (